tests/perf/components/telemetry/README.md

```diff
@@ -0,0 +1,32 @@
+# Telemetry Fluent Bit Perf Tests
+
+## Overview
+
+Small program to deploy a bunch of log pipelines to a Kubernetes cluster. The following pipelines are deployed:
+1. Simple log pipeline that logs to Loki
```

Comment:
I guess this should be "Single log pipeline ..."?

Reply:
True, will fix it